go/cmd/dolt/commands/status.go

```diff
@@ -43,19 +43,24 @@ func Status(ctx context.Context, commandStr string, args []string, dEnv *env.Dol
 	help, _ := cli.HelpAndUsagePrinters(commandStr, statusShortDesc, statusLongDesc, statusSynopsis, ap)
 	cli.ParseArgs(ap, args, help)
 
-	stagedDiffs, notStagedDiffs, err := actions.GetTableDiffs(ctx, dEnv)
+	stagedTblDiffs, notStagedTblDiffs, err := actions.GetTableDiffs(ctx, dEnv)
 
 	if err != nil {
 		panic(err) // fix
 	}
-
-	workingInConflict, _, _, err := actions.GetTablesInConflict(ctx, dEnv)
+	workingTblsInConflict, _, _, err := actions.GetTablesInConflict(ctx, dEnv)
 
 	if err != nil {
 		panic(err) // fix
 	}
 
-	printStatus(dEnv, stagedDiffs, notStagedDiffs, workingInConflict)
+	notStagedDcDiffs, err := actions.GetDocDiffs(ctx, dEnv)
+
+	if err != nil {
+		panic(err)
```

Comment:
Now is probably a good time to remove these panics.

Reply:
Got rid of panics: https://github.com/liquidata-inc/dolt/pull/222/commits/747a0a21ae8b387d9271c04d3046757b6e3c94ef